scripts/local_testnet/start_local_testnet.sh

```diff
@@ -1,147 +1,83 @@
 #!/usr/bin/env bash
-# Start all processes necessary to create a local testnet
 
-set -Eeuo pipefail
+# Requires `docker`, `kurtosis`, `yq`
 
-source ./vars.env
+set -Eeuo pipefail
 
-# Set a higher ulimit in case we want to import 1000s of validators.
-ulimit -n 65536
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+ENCLAVE_NAME=local-testnet
+NETWORK_PARAMS_FILE=$SCRIPT_DIR/network_params.yaml
 
-# VC_COUNT is defaulted in vars.env
-DEBUG_LEVEL=${DEBUG_LEVEL:-info}
-BUILDER_PROPOSALS=
+BUILD_IMAGE=true
+BUILDER_PROPOSALS=false
+CI=false
 
 # Get options
-while getopts "v:d:ph" flag; do
+while getopts "e:b:n:phc" flag; do
   case "${flag}" in
-    v) VC_COUNT=${OPTARG};;
-    d) DEBUG_LEVEL=${OPTARG};;
-    p) BUILDER_PROPOSALS="-p";;
+    e) ENCLAVE_NAME=${OPTARG};;
+    b) BUILD_IMAGE=${OPTARG};;
+    n) NETWORK_PARAMS_FILE=${OPTARG};;
+    p) BUILDER_PROPOSALS=true;;
+    c) CI=true;;
```

Comment:
I didn't include `DEBUG_LEVEL` and `VC_COUNT`, as they are already in `network_parameters.yaml` and is easy to modify if needed rather than adding a bunch of flags.

Some reasoning for the default values:
- `BUILD_IMAGE=true`: when testing locally, we usually want to test the code in working directory, so this is on by default. If anyone wants to use latest image, can also set this to false and update the image in `network_parameters.yaml` file
- `CI=false`: when testing locally, a block explorer and metrics explorer is usually handy but not useful on CI.

Reply:
I think the boolean flag may still be useful in the case of local testing, because if a user already have a local image and want to re-use it, then they could run it without rebuilding, and _sometimes_ they would want to rebuild.

My thoughts around designing the params is to prefer the use the `network_parameters.yaml` file where possible - this essentially replaces the `vars.env` - and avoid having config in two places for simplicity and not having to worry about priority. 

Same reason why I removed `DEBUG_LEVEL` and `VC_COUNT` from the scripts. I'm not a big fan of maintaining large shell scripts and would love to keep functionalities as small as possible.